rune/Rune.ts

```diff
@@ -96,6 +96,21 @@ export class Rune<out T, out U = never> {
     return value instanceof Rune ? value.into(ValueRune) : Rune.constant(value)
   }
 
+  static str<X, T extends TemplateStringsArray, R extends unknown[]>(
+    ...args: RunicArgs<X, [T, ...R]>
+  ) {
+    return Rune
+      .tuple(args)
+      .map(([strings, ...values]) =>
+        strings
+          .map((templateString, i) => {
+            const value = values[i]
+            return value !== undefined ? `${templateString}${value}` : templateString
```

Comment:
This `undefined` checking will have inconsistencies with regular template literals
```suggestion
            return i < values.length ? `${templateString}${values[i]}` : templateString
```